### DISCUSSION TITLE
Is there any way to whitelist introspection query from MaxQueryDepthInstrumentation?

### DISCUSSION THREAD

**Discussion by Aswin97** [graphql-java/graphql-java#2726] [Q&A]
When the introspection query is fired from GraphiQL, the following error is thrown
```
{
  "errors": [
    {
      "message": "maximum query depth exceeded 13 > 8",
      "extensions": {
        "classification": "ExecutionAborted"
      }
    }
  ]
}
```
Is there a workaround to whitelist the introspection query?

**Comment by bbakerman** [comment]
Out of the box we have no whitelisting - you could derive your own class and jump in at ` beginValidation(InstrumentationValidationParameters parameters) ` and use the Document etc... to find out the operation name or look at contained fields
